---
content_type: page
description: 'This section provides links to tools for molecular visualization and
  finding structures to view. '
hide_download: true
hide_download_original: null
learning_resource_types:
- Tools
ocw_type: CourseSection
title: Tools
uid: f0e682ad-7bb3-36c7-f223-70317318eb02
---

Molecular Visualization Software
--------------------------------

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
TOOLS
{{< thclose >}}
{{< thopen >}}
DESCRIPTIONS
{{< thclose >}}
{{< thopen >}}
WEB SITES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
DS Viewer Pro
{{< tdclose >}}
{{< tdopen >}}
One of the most user friendly programs is DS Viewer Pro, and is available for a free 30 days trial. This trial program is available for only Windows platforms.
{{< tdclose >}}
{{< tdopen >}}
[Download DS ViewerPro 5.0](https://discover.3ds.com/discovery-studio-visualizer-download)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Protein Explorer
{{< tdclose >}}
{{< tdopen >}}
Protein Explorer is another free modeling program that can be downloaded. Protein Explorer operates with Chime to provide a simple interface for novice users.
{{< tdclose >}}
{{< tdopen >}}
[FrontDoor to Protein Explorer 2.45 Beta](http://www.umass.edu/microbio/chime/pe_beta/pe/protexpl/vers_pe.htm?q=microbio/chime/explorer/vers_pe.htm)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Chime
{{< tdclose >}}
{{< tdopen >}}
Chime is based on the older RasMol, but more powerful and easier to use than the original. The Chime plugin, which is required to run Protein Explorer, can be downloaded.
{{< tdclose >}}
{{< tdopen >}}
[MDLÂ® Chime](https://www.umass.edu/microbio/chime/getchime.htm)
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}
  

If you would like to explore other program options, a comprehensive list of free molecular visualization and modeling software with brief descriptions can be found [here](http://web.archive.org/web/20101003180855/http://molvis.sdsc.edu/visres/molvisfw/titles.jsp).

Finding Structural Files
------------------------

Once you've chosen a macromolecule viewing program, you'll need to find structures to view!

To start out, try visiting [this page](http://www.umass.edu/microbio/chime/), which has an excellent collection of links to Chime biochemistry tutorials on many topics, including amino acids, protein structure, hemoglobin, enzymes, nucleic acids, lipid bilayers, and antibodies.

Protein Explorer is designed to be self-explanatory, so you should need only a little guidance to learn how to manipulate molecules. The [Quick-Start Protein Explorer](http://www.umass.edu/microbio/chime/pe/protexpl/frntdoor.htm) exercise, in conjunction with the [QuickTour guide](http://www.umass.edu/microbio/chime/explorer/qtour.htm), offers a nice introduction to the basic functions of Protein Explorer. Other resources include [FAQ](http://www.umass.edu/microbio/chime/explorer/faq.htm), a [glossary](http://www.umass.edu/microbio/chime/explorer/igloss.htm) of functions and terms, and a comprehensive [tutorial](http://www.umass.edu/microbio/chime/explorer/pe_tut.htm) for advanced use.

All structural coordinates (NMR and crystallographic) for biological macromolecules are deposited as PDB files in the [Brookhaven National Laboratory's Protein Data Bank (PDB)](http://www.rcsb.org/pdb/). You can locate specific files by using the search interface on the front page of that site. Once you identify an interesting file, click on "Explore" in the file's header, which will take you to a page with detailed information about the structure. From here, you can view the structure online or download the PDB file for viewing offline. [PDB Lite](http://www.rcsb.org/pdb/download/download.do) offers a simpler and more helpful interface for finding PDB files.

More than twenty other sources for PDB files, most of which are easier to browse than the main repository, are listed [here](http://web.archive.org/web/20101017043347/http://molvis.sdsc.edu/visres/pdbs/titles.jsp). For instance, on the PDBs for Biochemistry page you will find a nice collection of structures related to introductory biochemistry topics. [This site](http://nihlibrary.nih.gov/Pages/default.aspx) from the _Molecules R Us_ utility of the NIH allows you to input the name of a common molecule, such as aflatoxin, and get an easy to use structure in a few seconds.

Genomics and Proteomics Tools
-----------------------------

[ExPASy Proteomics Databases page](http://us.expasy.org/)

[BLAST at NCBI](http://www.ncbi.nlm.nih.gov/BLAST)

[NEB Cutter (open reading frame finder)](http://tools.neb.com/NEBcutter2/index.php)

[Entrez Nucleotide Sequence Database (aka GenBank)](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Nucleotide)

[Entrez Protein Sequence Database](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Protein)

[Protein Data Bank of Structures](http://www.rcsb.org/pdb/)

[O Glycosylation Site Search](http://www.cbs.dtu.dk/services/NetOGlyc/)

[N Glycosylation Site Search](http://www.cbs.dtu.dk/services/NetNGlyc/)

[Phosphorylation Site Search](http://www.cbs.dtu.dk/services/NetPhos/)

[Swiss-Prot and TrEMBL](http://us.expasy.org/sprot/)

[Prosite](http://us.expasy.org/tools/scanprosite/)